contentpacks/__main__.py

```diff
@@ -46,6 +46,7 @@ def make_language_pack(lang, version, sublangargs, filename, ka_domain, no_asses
     all_assessment_data, all_assessment_files = retrieve_all_assessment_item_data(
         no_item_data=no_assessment_items,
         no_item_resources=no_assessment_resources,
+        node_data=node_data,
```

Comment:
@cpauya & @mrpau-eduard this commit will fix the Swahili assessment item issue. I tested it in my local.